base/errorshow.jl

```diff
@@ -32,9 +32,25 @@ showerror(io::IO, ex) = show(io, ex)
 show_index(io::IO, x::Any) = show(io, x)
 show_index(io::IO, x::Slice) = show_index(io, x.indices)
 show_index(io::IO, x::LogicalIndex) = summary(io, x.mask)
-show_index(io::IO, x::OneTo) = print(io, "1:", x.stop)
+function show_index(io::IO, x::OneTo)
+    if x.stop == 1
```

Comment:
Why not consistently print as `start:stop` no matter then length like before? Also for `Tuple` below.

Reply:
Maybe. Just a style choice but it didn't really matter to me. I'll fix.

(Note to self: Strings)